backend/papersfeed/utils/collections/utils.py

```diff
@@ -152,6 +153,65 @@ def select_collection_user(args):
     return collections
 
 
+def update_paper_collection(args):
+    """Update Paper Collection"""
+    is_parameter_exists([
+        constants.ID, constants.COLLECTION_IDS
+    ], args)
+
+    # Request User
+    request_user = args[constants.USER]
+
+    # Paper Id
+    paper_id = args[constants.ID]
+
+    # Collection IDs
+    collection_ids = json.loads(args[constants.COLLECTION_IDS])
+
+    # Containing Collections
+    containing_collection_ids = __get_collections_contains_paper(paper_id, request_user)
+
+    # Add To Collections
+    __add_paper_to_collections(paper_id, list(set(collection_ids) - set(containing_collection_ids)))
+
+    # Remove From Collections
+    __remove_paper_from_collections(paper_id, list(set(containing_collection_ids) - set(collection_ids)))
+
+
+def __get_collections_contains_paper(paper_id, request_user):
+    """Get Collections Containing paper"""
+    collections = CollectionUser.objects.filter(
```

Comment:
If you use ManyToManyField, many codes like this part would be unnecessary or shorter, and faster.